src/nft/components/profile/list/FloatingBanner.tsx

```diff
@@ -0,0 +1,112 @@
+import { buttonHoverState } from 'components/Common'
+import { getTotalEthValue } from 'nft/components/bag/profile/utils'
+import { useSellAsset } from 'nft/hooks'
+import { Listing, WalletAsset } from 'nft/types'
+import { formatUsdPrice } from 'nft/utils/currency'
+import { fetchPrice } from 'nft/utils/fetchPrice'
+import { useEffect, useMemo, useState } from 'react'
+import styled from 'styled-components/macro'
+import { ThemedText } from 'theme'
+
+const FloatingBannerGradientContainer = styled.div`
```

Comment:
Is the banner supposed to be the same width as and centered on the grid?
![Screen Shot 2022-12-09 at 08 02 12 ](https://user-images.githubusercontent.com/11512321/206742779-f705cd36-560c-442e-922e-47918a175671.png)

Reply:
Gradient also is not full desktop width